src/openapi/streaming/streaming.ts

```diff
@@ -143,6 +143,7 @@ class Streaming extends MicroEmitter<EmittedEvents> {
     };
     reconnecting = false;
     contextId!: string;
+    contextMessageCount!: number;

```

Comment:
Not sure what to call this...

underlying connection message count - what it really is
connection message count - misleading because the Connection class is long-lived
session message count - what OpenAPI calls it